src/pages/PasswordChange.jsx

```diff
@@ -0,0 +1,100 @@
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { Component } from 'react';
+import { Button } from '../components/Buttons/Button/Button';
+import { setUserValidation, isCorrectPassword } from '../services/services';
+import firebase from '../services/firebase';
+
+class PasswordChange extends Component {
+  constructor(props) {
+    super();
```

Comment:
You should be careful. Call `super(props)`  or do not do it

Reply:
yes, you right. Thx.
```suggestion
    super(props);
```